docker-compose.yml

```diff
@@ -45,27 +45,40 @@ services:
       - ./appeals-service-api:/opt/app
     command: npm run start:dev
 
-  document-service-api:
+  document-service-api: &document-service
```

Comment:
Didn't knew about this feature :o

Reply:
Called "YAML Anchors". Very useful https://blog.daemonl.com/2016/02/yaml.html